src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -145,6 +157,24 @@ def _get_current_exposure(self, index: pd.Index) -> pd.Series:
             if mam_coverage == "baseline":  # return standard exposure if baseline
                 propensity = self.propensity(index)
                 return pd.Series(self.exposure_distribution.ppf(propensity), index=index)
+            elif mam_coverage == "targeted":
+                # initialize exposures as 0s using index
+                exposures = pd.Series(0, index=index)
+
+                # define relevant booleans
+                wasting = self.wasting_exposure(index)
+                age = self.population_view.get(index)["age"]
+                underweight = self.underweight_exposure(index)
+
+                in_mam_state = wasting == "cat2"
+                in_age_range = (age >= 0.5) & (age < 2)
+                is_severely_underweight = underweight == "cat3"
```

Comment:
I know underweight risk categories work differently from wasting and stunting. Just want to confirm that `cat3` is the correct category here for severely underweight.

Reply:
Yeah I checked the artifact categories. Would it be worth storing these as constants?